lib/s3-service.js

```diff
@@ -11,7 +11,7 @@ const { getSignedUrl } = require('@aws-sdk/s3-request-presigner');
 const { streamToBuffer } = require('./utils');
 
 // Export
-exports.client = new S3Client();
+exports.setClient = (config) => (exports.client = new S3Client(config));
 
```

Comment:
Imo, in order to not making a breaking change,
we should not remove the ``exports.client = new S3Client()``` above.

Wdyt?

Also, regarding naming of setClient,
maybe we should rename to setConfig or overrideConfig because we're not passing a client, wdyt?

Reply:
Good points, totally agree. I have made updates to reflect this 😄